### PR TITLE
added cscorelib version to the guide

### PR DESCRIPTION
**Pull request by J3fftw1**
## Description
added cscorelib to sf guide info panel.
people who have no op can now view the versions to make a proper report
also fixed occurred

## Changes
added cscorelib version to the guide

## Related Issues
NA

## Testability

- [x] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [x] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.

### CODE REVIEW

File: src/main/java/me/mrCookieSlime/Slimefun/SlimefunGuide.java [hunk 1]
```diff
@@ -143,7 +144,15 @@ else if (SlimefunManager.isItemSimiliar(guide, getItem(SlimefunGuideLayout.CHEAT
 			});
 		}
 
-		menu.addItem(1, new CustomItem(new ItemStack(Material.WRITABLE_BOOK), "&aCredits", "", "&7Version: &a" + SlimefunPlugin.instance.getDescription().getVersion(), "&7Contributors: &e" + SlimefunPlugin.getUtilities().contributors.size(), "", "&7\u21E8 Click to see the people behind this Plugin"));
+		menu.addItem(1, new CustomItem(new ItemStack(Material.WRITABLE_BOOK),
+				"&aCredits",
+				"",
+				"&7Slimefun Version: &a" + SlimefunPlugin.instance.getDescription().getVersion(),
+				"&7CSCore-Lib Version: &a" + CSCoreLib.getLib().getDescription().getVersion(),
```
Comment (TheBusyBiscuit) [inline src/main/java/me/mrCookieSlime/Slimefun/SlimefunGuide.java]: You placed the hyphon at the wrong spot.

And I purposefully didn't include that as CS-CoreLib2 is used in most classes now, so we can correctly identify the CS-CoreLib version based on the Slimefun version.

CS-CoreLib itself will be faded out as a dependency and is not needed for future bug reports, even now it shouldn't be a source of error anymore.

Comment (J3fftw1) [reply src/main/java/me/mrCookieSlime/Slimefun/SlimefunGuide.java]: we still require it for bug reports though
and people that use older version are having bugs with it for example the ancient altar bugs on older versions

Comment (TheBusyBiscuit) [reply src/main/java/me/mrCookieSlime/Slimefun/SlimefunGuide.java]: We don't support older versions anyway...

Comment (J3fftw1) [reply src/main/java/me/mrCookieSlime/Slimefun/SlimefunGuide.java]: i mean older versions of CS-CoreLib

**Comment by WalshyDev** [comment]
and merged 😂

**Pull request merged**